# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1427037507 -3600
#      Sun Mar 22 16:18:27 2015 +0100
# Node ID f902855f870ac4170eeee625d1e91b5a1d5811f5
# Parent  0f2cbcb14a21372ac696341f41d6cf7cef93e425
Fix right click zoom in fltk after cset f134ee404041 (bug #44522)

* __init_fltk__.cc (plot_window::handle): replace "alternate", which is not an allowed value for "selectiontype" after cset f134ee404041, by "alt"

diff --git a/libinterp/dldfcn/__init_fltk__.cc b/libinterp/dldfcn/__init_fltk__.cc
--- a/libinterp/dldfcn/__init_fltk__.cc
+++ b/libinterp/dldfcn/__init_fltk__.cc
@@ -1455,17 +1455,17 @@ private:
               set_currentpoint (pos_x, pos_y);
 
               if (Fl::event_button () == FL_LEFT_MOUSE
                   && Fl::event_shift ())
                 fp.set_selectiontype ("extend");
               else if ((Fl::event_button () == FL_LEFT_MOUSE
                         && Fl::event_ctrl ())
                        || Fl::event_button () == FL_RIGHT_MOUSE)
-                fp.set_selectiontype ("alternate");
+                fp.set_selectiontype ("alt");
               else if (Fl::event_clicks ())
                 fp.set_selectiontype ("open");
               else
                 fp.set_selectiontype ("normal");
 
               gh = pixel2axes_or_ca (pos_x, pos_y);
 
               if (gh.ok ())
